tests/bin/cloud_health_to_graphite_test.py

```diff
@@ -216,6 +233,40 @@ def test_run_with_set_date(self, mock_stdout):
 
         self.assertEqual(prints, mock_stdout.getvalue())
 
+    @patch('sys.argv', ['prog', API_KEY, REPORT_ID_ARG, '--report-name', REPORT_NAME_ARG, '--interval', INTERVAL.name])
+    @patch('sys.stdout', new_callable=StringIO)
+    def test_run_with_interval(self, mock_stdout):
+        """
+        Cloud Health to Graphite: The date in the data is correctly parsed with the passed --interval
+        """
+        app = Application()
+        app.cloud_health.get_custom_report = MagicMock(side_effect=CloudHealthAPITest._get_cloud_health_return)
+
+        app.run()
+
+        prints = ''
+
+        api_data = CloudHealthAPITest._get_cloud_health_return(
+            self.REPORT_ID, time_interval=self.INTERVAL,
+        )
+        for date, values in iteritems(api_data):
+            if date is 'Total':
+                continue
+
+            date = int(calendar.timegm(datetime.strptime(date, self._DEFAULT_DATE_FORMAT).utctimetuple()))
```

Comment:
You've checked if `date is 'Total'`, what about other invalid values? I do not know a lot about the context, but it might be better idea to check acceptable lines with some regex. But if you 100% sure there will be no other invalid values it looks fine.